src/main/webapp/login.html

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Login Page</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="script.js"></script>
+  </head>
+  <body>
+    <form onsubmit="storeZipCode();" method="POST">
+        <label for="zipcode">Zipcode: </label>
+        <input type="text" id="zipcode" name="zipcode" value="80124"><br><br>
```

Comment:
Is this value the default value? Before they enter anything?

Reply:
maybe just make it a bunch of zeroes?

Reply:
yep the default - changed to 00000